README.md

```diff
@@ -11,14 +11,22 @@
 
```

Comment:
Can we put these warnings to the end? Not really inviting if the first piece of Info we see is something like "WARNING, UNFINISHED, will spoil your milk if you use it..."